pokecli.py

```diff
@@ -117,7 +117,7 @@ def init_config():
         "Set the unit to display distance in (e.g, km for kilometers, mi for miles, ft for feet)",
         type=str,
         default="km")
-    
+
```

Comment:
only have line change?
The commit may be merged, Let me close it here. Reopen when you need it .